dlang.org.ddoc

```diff
@@ -438,8 +438,12 @@ _=
 
 SUBNAV_TOOLS=
 $(SUBNAV_TEMPLATE
-    $(SUBNAV_HEAD D-Specific Tools, $(ROOT_DIR)tools.html, overview)
+    $(SUBNAV_HEAD Command-line Reference, $(ROOT_DIR)tools.html, overview)
     $(UL $(SUBMENU2
+        $(ROOT_DIR)dmd-windows.html, dmd (Windows),
+        $(ROOT_DIR)dmd-linux.html, dmd (Linux),
+        $(ROOT_DIR)dmd-osx.html, dmd (OSX),
+        $(ROOT_DIR)dmd-freebsd.html, dmd (FreeBSD),
         https://github.com/Hackerpilot/dfix, dfix,
         https://github.com/Hackerpilot/dfmt, dfmt,
```

Comment:
How about replacing these with a link to the tools page on the wiki, at the bottom of the list?